pkg/integration/components/test.go

```diff
@@ -45,6 +48,55 @@ type NewIntegrationTestArgs struct {
 	ExtraCmdArgs string
 	// for when a test is flakey
 	Skip bool
+	// to run a test only on certain git versions
+	GitVersion GitVersionRestriction
+}
+
+type GitVersionRestriction struct {
+	// Only one of these fields can be non-empty; use functions below to construct
+	from     string
+	before   string
```

Comment:
But I don't _want_ them to be consistent (or symmetric), because one is inclusive and the other is not. From/To or Before/After don't express that, but From/Before does (or at least they do to me), and I think they read quite naturally when used in tests. If you don't like them, I'd rather go back to GreatThanOrEqual/LessThan.